apps/shuffle/butterfly_network.py

```diff
@@ -75,7 +75,8 @@ def getNTriplesAndSbits(tripleshares, randshares, n):
     # k (log k)^2
     assert k == len(inputs)
     assert k & (k-1) == 0, "Size of input must be a power of 2"
-    benchLogger = BenchmarkLogger.get(ctx.myid)
+    benchLogger = logging.LoggerAdapter(
```

Comment:
Sucks to have two-line + indent log statements, it's a discouragement from using them :( Can we simplify it with an import/def up top?